spices/bin/page-plugins/CWD.js

```diff
@@ -0,0 +1,7 @@
+const CWD = function () {
```

Comment:
I am almost certain you can use `process` and `__dirname` inside pug. Need to double check.

Reply:
100%. This is purely a testing plugin.

Reply:
Gotcha!